Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/transform/subdisc/HL7ArrayTransforms.java

```diff
@@ -214,13 +217,13 @@ public static MFMIMT700701UV01AuthorOrPerformer copyNullFlavors(QUQIMT021001UV01
         return to;
     }
 
-    public static PRPAIN201301UV02 copyMCCIMT000100UV01AttentionLine(PRPAIN201306UV02 from, PRPAIN201301UV02 to) {
+    public static PRPAIN201301UV02 copyMCCIMT000100UV01AttentionLine(final PRPAIN201306UV02 from, PRPAIN201301UV02 to) {
```

Comment:
Just following up on this note: the "to" param can't automatically be converted to final because existing code reassigns it.  I have this (along with tons of other param reassignment issues) taken care of as part of a SonarQube fix branch:

https://github.com/CONNECT-Solution/CONNECT/commit/fe4e55fb0a309d835c65f61d21e7135723391afa?w=1#diff-9b46758c7040be921819a3af60986c57R228